lib/generator.js

```diff
@@ -305,6 +305,7 @@ class Generator {
       npmi({
         name: this.templateName,
         install: force,
+        path: ROOT_DIR,
```

Comment:
To clarify: ROOT_DIR is the directory where the library is currently installed. It's not the system's root dir 😄

Reply:
makes sense and `npmi` default is `path: '.'`